cpp/cmake_modules/FindSnappy.cmake

```diff
@@ -15,25 +15,33 @@
 # specific language governing permissions and limitations
 # under the License.
 
+set(SNAPPY_STATIC_LIB_SUFFIX "${SNAPPY_MSVC_STATIC_LIB_SUFFIX}${CMAKE_STATIC_LIBRARY_SUFFIX}")
```

Comment:
Is this needed?
It seems that Snappy doesn't add any suffix for MSVC: https://github.com/google/snappy/blob/master/CMakeLists.txt

Reply:
Thanks for double-checking.